tests/gui/os/window-progress.spec.js

```diff
@@ -62,6 +64,16 @@ describe('Browser: WindowProgress', function() {
           }).to.throw('Invalid percentage: -1');
         });
 
+        it('should set the flashing title', function() {
+          windowProgress.set(100, 'write');
+          m.chai.expect(this.setTitleSpy).to.have.been.calledWith('Etcher – Flashing 100%');
+        });
+
+        it('should set the validating title', function() {
+          windowProgress.set(100, 'check');
+          m.chai.expect(this.setTitleSpy).to.have.been.calledWith('Etcher – Validating 100%');
+        });
+
```

Comment:
Can we also have a test that after Validation, the title gets set back to just "Etcher" ?